### PR TITLE
fix possible divided by zero error

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: api/src/main/java/io/minio/PutObjectArgs.java [hunk 1]
```diff
@@ -93,11 +93,8 @@ private void validatePartCount(int partCount, long objectSize, long partSize) {
       }
 
       if (partSize > 0) {
-        if (partSize > objectSize) {
-          partSize = objectSize;
-        }
-
-        long partCount = (long) Math.ceil((double) objectSize / partSize);
+        if (partSize > objectSize) partSize = objectSize;
+        long partCount = partSize > 0 ? (long) Math.ceil((double) objectSize / partSize) : 1;
```
Comment (kannappanr) [inline api/src/main/java/io/minio/PutObjectArgs.java:97]: partsize is always greater than 0 here. Please look at line 95. So, not sure why this change is needed.

Comment (balamurugana) [reply api/src/main/java/io/minio/PutObjectArgs.java:97]: line 96 may make it zero.

**Review by kannappanr — APPROVED** [review]
